### ISSUE TITLE
[Mob] Casting while moving

### ISSUE THREAD

**Issue by Rumsteak** [dalaranwow/dalaran-wow#2473]
Im not 100% sure if its working as intented or a bug, but i dont remember it was working that way.

Zone: Northern Barrens
Mob: Kolkar Stormer
Wowhead: http://www.wowhead.com/npc=3273/kolkar-stormer

They cast their lightning bolt and lightning cloud while moving in about 50% of the cases, tested like 10 times.

**Comment by InvincibleEagle** [comment]
I've had similar problems with burning blade mobs in Desolace continuing to patrol, aggroing more mobs as they continued to cast spells at me.

**Comment by warcraftgeekygamer** [comment]
I've experienced this with several creatures that have the ability to cast spells. Creatures in Redridge (the Gnolls) could cast spells while moving. Creatures in Arathi Highlands could cast spells while moving. I do not believe this is something zone related, nor is it creature specific. I believe this is happening to any type of creature that can cast spells (ie: shadowbolt, lightning bolt, healing wave, etc).

Bottom line.....creatures that can cast spells while moving should NOT be happening.

**Comment by nowonshere** [comment]
Encountered the same bug with some mobs in Northrend, e.g. Anub'ar Cultists in Icemist Village, casters in Moonrest Gardens.

**Comment by joelmatheny** [comment]
Confirmed again - all caster mobs continue moving while casting. This should not happen, they should stand in place when they start casting.

**Comment by Rushor** [comment]
can you list the ids of the creature?

**Comment by hAkonf** [comment]
Bosses do the same.

**Comment by Rushor** [comment]
DB/SAI: Kolkar Stormer - Add CMC

* closes https://github.com/dalaranwow/dalaran-wow/issues/2473
* Kolkar Stormer  can no longer cast "Lightning Bolt" while moving
* commit: dc61616a383965291a56173b1a32fcecd6dca9b7

If this issue also affects more creatures please open up an new issue with the id from http://wotlk.openwow.com/. ty

The issue is fixed on the PTR Realm. The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Please see and watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.

**Comment by TeridanDW** [comment]
Works as intended on ptr.
